src/tests/ftest/util/test_utils_pool.py

```diff
@@ -254,6 +256,30 @@ def destroy(self, force=1):
 
         return status
 
+    @fail_on(DaosApiError)
+    def disable_aggregation(self):
+        """Disable Aggregation.
+
+        It disables aggregation for a given pool uuid using
+        dmg.
+
+        """
+        if self.pool:
```

Comment:
Given that this method is exclusive to the dmg method of issuing comands, there should be a check and error reporting for the existance of `self.dmg`.

Reply:
Resolving as this was already in the code.